modules/lib/maintainers.nix

```diff
@@ -101,6 +101,12 @@
     githubId = 1204734;
     name = "Emil Karlson";
   };
+  johnrtitor = {
+    email = "masumrezarock100@gmail.com";
+    github = "johnrtitor";
+    githubId = 50095635;
+    name = "Masum Reza";
+  };
```

Comment:
You already seem to be present in the Nixpkgs maintainers list, in which case you don't need to add yourself here as well.

```suggestion
```

Reply:
Done, thanks :)